src/testdir/test_options.vim

```diff
@@ -382,6 +386,278 @@ func Test_set_completion()
   call assert_equal('"set syntax=' .. getcompletion('a*', 'syntax')->join(), @:)
 endfunc
 
+" Test handling of expanding individual string option values
+func Test_set_completion_string_values()
```

Comment:
Most of the `assert_equal()` calls in this function have wrong order of arguments. This will make error message confusing if test fails.

Reply:
Oh right… the first item is the expected value. I internalized the weirdness when I was debugging this but didn't correct it. I could fix it in a follow-up commit.